docs/accessibility.md

```diff
@@ -0,0 +1,9 @@
+# Accessibility
+
+The **Microsoft MakeCode** team wants to be sure that every person on the planet can play with it to achieve more. It is very important for us when we're working to make computer science more accessible to consumers with disabilities.
+
```

Comment:
Add high level description of the accessibility strategy: menu, javascript accessible, blocks not accessible, color contrasts, high contrast.